mongocrypt-sys/Cargo.toml

```diff
@@ -0,0 +1,21 @@
+[package]
+name = "mongocrypt-sys"
+description = "FFI bindings to libmongocrypt"
+version = "0.1.0"
```

Comment:
one thing some sys crates do is add the version of the library it matches to the suffix, e.g. `0.1.0+1.5.0-rc0`. I'm not sure if that's just a thing for sys crates that optionally vendor in or if it's more general though. Only reason I bring it up is that I tried to build this with 1.3.0 and it failed, so having the version number here might indicate better what our min requirement is. (I'm also now seeing that you included this in the README :facepalm: )

Reply:
Makes sense, done.